cogniac/edgeflow.py

```diff
@@ -5,22 +5,25 @@
 """
 
 import os
+import re
+from time import time
 import six
-import sys
 import requests
-import re
 from retrying import retry
 from requests.packages.urllib3 import Retry
 from requests.adapters import HTTPAdapter
-from time import time
 
 from .common import server_error, raise_errors
-
 from .media import file_creation_time
 
+from pprint import pprint
```

Comment:
is this used?

Reply:
left over from some testing - I will remove.